src/test/java/org/prebid/server/bidder/gamma/GammaBidderTest.java

```diff
@@ -0,0 +1,328 @@
+        assertThat(result.getErrors().get(0).getType()).isEqualTo(BidderError.Type.bad_server_response);
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnEmptyResultWhenResponseWithNoContent() {
+        // given
+        final HttpCall<Void> httpCall = HttpCall.success(null, HttpResponse.of(204, null, null), null);
+
+        // when
+        final Result<List<BidderBid>> result = gammaBidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnVideoWhenBidRequestImpIsVideo() throws JsonProcessingException {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder().id("id").video(Video.builder().build()).build()))
+                .build();
+        final HttpCall<Void> httpCall = givenHttpCall(mapper.writeValueAsString(
+                BidResponse.builder()
+                        .id("id")
+                        .seatbid(singletonList(SeatBid.builder()
+                                .bid(singletonList(Bid.builder().build()))
+                                .build()))
+                        .build()));
+
+        // when
+        final Result<List<BidderBid>> result = gammaBidder.makeBids(httpCall, bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue())
+                .containsOnly(BidderBid.of(Bid.builder().build(), video, "USD"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnBannerWhenNoProvided() throws JsonProcessingException {
+        // given
+        final BidRequest bidRequest = BidRequest.builder().imp(emptyList()).build();
+        final HttpCall<Void> httpCall = givenHttpCall(mapper.writeValueAsString(
+                BidResponse.builder()
+                        .id("id")
+                        .seatbid(singletonList(SeatBid.builder()
+                                .bid(singletonList(Bid.builder().build()))
+                                .build()))
+                        .build()));
+
+        // when
+        final Result<List<BidderBid>> result = gammaBidder.makeBids(httpCall, bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue())
+                .containsOnly(BidderBid.of(Bid.builder().build(), banner, "USD"));
+    }
+
+    @Test
+    public void extractTargetingShouldReturnEmptyMap() {
+        assertThat(gammaBidder.extractTargeting(mapper.createObjectNode())).isEqualTo(emptyMap());
+    }
+
+    private static HttpCall<Void> givenHttpCall(String body) {
+        return HttpCall.success(
+                HttpRequest.<Void>builder().build(),
+                HttpResponse.of(200, null, body),
+                null);
+    }
+}
```

Comment:
Can't recall if we need an empty line in tests, but lets add one here while we are at it)